website/data/pages/radius-&-shadows.mdx

```diff
@@ -1,3 +1,34 @@
 ---
 title: Radius & shadows
----
\ No newline at end of file
+description:
+---
+
+# Radius & shadows
+
+## Border Radius
+
+To optimize the global consistency, all elements with a corner radius are standardized at 4px.
+
+![Border radius](/border-radius.png)
+## Examples:
+**‣**  Buttons <br/> <br/>
+**‣**  Filters <br/> <br/>
+**‣**  Tables <br/> <br/>
+**‣**  Content container <br/> <br/>
+**‣**  Dropdown <br/> <br/>
+
+## Drop shadows
+
+In order to keep consistency across the product, the shadows also follow certain rules.
+
+![Drop shadows](/drop-shadows.png)
+We use it on 3 differents cases : <br/> <br/>
+**‣**  On the content containers (in addition with the border below) <br/>
+**‣**  On the Filters <br/> 
+**‣**  On the PopOvers & Modals <br/> 
+
+## Table & Content container border
+
+In addition to the related drop shadow, we add a slight border on the main container, PopOvers, Modals, HelpBox and every other elements we want to highlight or make pop over the interface.
+
+![Table & Content container Border](/table-and-content-container.png)
```

Comment:
We don't have carbon colors

Reply:
Replaced it with **Neutral 150**, thanks